src/hooks/screen/useCreateChat.ts

```diff
@@ -124,6 +125,21 @@ const useCreateChat = () => {
     }
   };
 
+  const createAnonymousChat = async (selectedUser) => {
+    try {
+      setLoadingCreateChat(true);
+      const response = await getOrCreateAnonymousChannel(selectedUser?.user?.userId);
+      const chatData = createChannelJson(response, selectedUser);
+      const channelList = ChannelList.fromMessageAnonymouslyAPI(chatData);
+      await channelList.saveIfLatest(localDb);
+      handleMemberSchema(response);
+      goToChatScreen(channelList, GROUP_INFO);
+    } catch (e) {
+      setLoadingCreateChat(false);
```

Comment:
mas @GarryLaly consider moving the setLoadingCreateChat in `finally` block.
```
try {
    setLoadingCreateChat(true)
    ....
} catch(e) {
   console.error(...)
} finally {
   setLoadingCreateChat(false)
}
```

Reply:
Fixed